src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, e => {
+      resolve(`It was ${eventName} on the element:
+        ${element.nodeName}, id: ${element.id}.`);
+    });
+  });
 }
 
 const printMessage = (message) => {
-// write your code here
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = `${message}`;
```

Comment:
Wouldn`t it works as well?
```suggestion
  div.textContent = message;
```